include/seqan3/argument_parser/detail/format_base.hpp

```diff
@@ -78,6 +78,66 @@ class format_base
             return detail::get_display_name_v<value_type>.str();
     }
 
+    /*!\brief Returns the corresponding GKN type for the input type.
+     *
+     * \tparam value_type The type whose name is converted std::string.
+     * \tparam validator_type The type of the validator object required for distinguishing 
+     * strings from input/output file types.
+     *
+     * \return The type of the value as a string.
+     */
+    template<typename value_type, typename validator_type>
+    static std::string get_type_as_gkn_string(value_type const & /* option */,
```

Comment:
```suggestion
    static std::string get_type_as_ctd_string(value_type const & /* option */,
```

Reply:
Yes, the name you suggest is more appropriate